content/docs/2.1/scalers/mongo.md

```diff
@@ -0,0 +1,80 @@
+# mongo with ScaledJob
```

Comment:
Can you add the typical scaler header please?

https://github.com/kedacore/keda-docs/blob/master/content/docs/2.1/scalers/azure-service-bus.md#L1-L8

Feel free to drop the reference to `ScaledJob`